xml/System.Text.Json.Serialization/JsonAttribute.xml

```diff
@@ -31,9 +31,9 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Creates a new instance of the <see cref="T:System.Text.Json.Serialization.JsonAttribute" /> instance.</summary>
```

Comment:
Do we generally repeat `instance` twice like this?

`Create an instance of the <FOO> instance.`

Reply:
Maybe:

```suggestion
        <summary>Creates a new instance of the <see cref="T:System.Text.Json.Serialization.JsonAttribute" />.</summary>
```

Also, JsonAttribute is an abstract class. What ctor?

https://github.com/dotnet/corefx/blob/ba2d62a24ea8574f40c43cd27cc384f079958a0c/src/System.Text.Json/src/System/Text/Json/Serialization/JsonAttribute.cs#L10

I guess the auto-generated protected ctor. Is that the general pattern for abstract classes?

https://github.com/dotnet/corefx/blob/ba2d62a24ea8574f40c43cd27cc384f079958a0c/src/System.Text.Json/ref/System.Text.Json.cs#L453-L456

Reply:
This is a very good question. Why did a protected constructor get added by mdoc to the xml structure? @mairaw is this a bug? I don't think we should be documenting protected elements.

Reply:
>  I don't think we should be documenting protected elements.

Sure we should.  If you override the type you need to know what the protected members do...

Reply:
I'll commit it but leave the conversation unresolved.

Reply:
https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Constructor says the formula for a constructor in an abstract class is `Called from constructors in derived classes to initialize the <Class> class.`